balancer/base/balancer.go

```diff
@@ -28,6 +28,8 @@ import (
 	"google.golang.org/grpc/resolver"
 )
 
+var logger = grpclog.Component("CORE")
```

Comment:
Should we rename "CORE" to just "gRPC"?  I get that these are "gRPC logs" but they could be getting mixed in with the application's non-gRPC logs, so it's not necessarily redundant and could help.  @menghanl ?

Reply:
I'm not sure. I'm OK with `core`.

How would users tell gRPC logs from other application logs? They set the gRPC logger to something that always adds a `grpc` prefix?

Reply:
> How would users tell gRPC logs from other application logs? They set the gRPC logger to something that always adds a grpc prefix?

Presumably?  I can't see any other way beyond having separate log files.  My guess is most don't do either, but it would just be a guess.

Reply:
If they do `grpc` prefix, making this grpc would be redundant.
If they don't, they won't know the other components are grpc logs because they might not have the grpc keyword.

Should we prepend `grpc` to all grpc logs?

Reply:
It's not terribly redundant.  The name of the component could be considered "grpc" instead of "core".

> Should we prepend grpc to all grpc logs?

Maybe if we had always done it.  But at this point, I'd say no.  This is definitely redundant with the fact that it's coming from the grpclog package.